pipeline/README_ONCOANALYSER.md

```diff
@@ -75,58 +78,53 @@ Require inputs shown as :white_check_mark: for available analyses
 
 Require inputs shown as :white_check_mark: for available analyses
 
-| Analysis name | Tumor DNA Fastq or BAM | Tumor RNA Fastq or BAM |
-| ---           | :-:                | :-:           |
-| Tumor only    | :white_check_mark: | *optional*    |
+| Analysis name | Tumor DNA (FASTQ/BAM) | Tumor RNA (FASTQ/BAM) |
+| ------------- | :-------------------: | :-------------------: |
+| Tumor only    |  :white_check_mark:   |      _optional_       |
 
 ## Usage
 
 ### Software requirements
 
-* Nextflow >=22.10.5 ([instructions](https://www.nextflow.io/index.html#GetStarted))
-* Docker ([instructions](https://docs.docker.com/engine/install/#server))
+- Nextflow >=22.10.5 ([instructions](https://www.nextflow.io/index.html#GetStarted))
+- Docker ([instructions](https://docs.docker.com/engine/install/#server))
 
 > [!NOTE]
 > Docker on Windows and macOS can perform poorly, so only running oncoanalyser on Linux is currently recommended.
 
-> [!WARNING]
-> Older versions of Docker may not work with Oncoanalyser. We recommend Docker version 25 or later. If you are
-> running an older version of your operating system, you may need to upgrade it in order to use an up to date
-> version of Docker.
```

Comment:
I removed this since I wasn't sure whether it was Docker versions or host kernel versions causing the incompatibility. Docker is meant to have fairly solid backwards compatibility.

@mkcmkc are you able to confirm? If you feel strongly about keeping this in, no worries! I'd prefer to only have it in if it was definitely the issue, but I'll leave the decision to you